src/main/java/seedu/multitasky/storage/StorageManager.java

```diff
@@ -14,6 +14,7 @@
 import seedu.multitasky.model.ReadOnlyEntryBook;
 import seedu.multitasky.model.UserPrefs;
 
+//@@author A0132788U
```

Comment:
The collator for the code takes everything from the //@@author onwards until it finds another //@@author, so if you have one at the beginning of the file there's no need for the others.